features/main-ui.feature

```diff
@@ -44,3 +44,11 @@ Feature: Main UI
     When There is no wallet stored
     And I am on the Daedalus Transfer instructions screen
     Then I see transactions buttons are disabled
+
+  @it-110
+  Scenario: Ensure user can hide balance (IT-110)
```

Comment:
I think we are missing the test that we are able to see the balance

Reply:
oh you mean after hiding?  I'm on it.

Reply:
yeah, so we make sure that the update code to display the value is okay